slack_today_i_did/our_repo.py

```diff
@@ -29,7 +30,20 @@ def _git_init(self) -> None:
     def _git_clone(self, branch_name: str = 'master') -> None:
         os.system(f'git remote set-branches origin {branch_name}')
         os.system(f'git fetch --depth 1 origin {branch_name}')
-        os.system(f'git checkout origin/{branch_name}')
+        os.system(f'git checkout {branch_name}')
+
+    def _git_file_on_branch(self, filename: str, branch_name: str = 'master') -> str:
+        current_dir = os.getcwd()
+        try:
+            os.chdir(self.repo_dir)
+            filename = filename.lstrip(self.repo_dir + '/')
+            git_log = f"git checkout {branch_name} {filename} && cat {filename}"
+            output = subprocess.check_output(git_log, shell=True)
+
+            os.system(f'git checkout master {filename}')
```

Comment:
:art: equivalent to `subprocess.call(f'git checkout master {filename}', shell=True)`